era_5g_server/BUILD

```diff
@@ -9,7 +9,7 @@ python_distribution(
     sdist=True,
     provides=setup_py(
         name="era_5g_server",
-        version="0.4.0",
+        version="0.5.0",
```

Comment:
This is just a bug fix. It doesn't break compatibility with anything. So it should be `0.4.1`.

Reply:
I am not sure, the disconnect function (NetworkApplicationServer) was also added in previous commit.

Reply:
Sure, that's true, I didn't realize.